Src/dronecan_application/modules/circuit_status/CircuitStatusModule.cpp

```diff
@@ -6,26 +6,15 @@
 #include "CircuitStatusModule.hpp"
 
 
-CircuitStatusModule CircuitStatusModule::instance = CircuitStatusModule();
-bool CircuitStatusModule::instance_initialized = false;
-Logger CircuitStatusModule::logger = Logger("CircuitStatus");
-
-CircuitStatusModule& CircuitStatusModule::get_instance() {
-    if (!instance_initialized) {
-        if (instance.init() != 0) {
-            logger.log_debug("ADC init error");
-        } else {
-            instance_initialized = true;
-        }
-    }
-    return instance;
-}
+bool CircuitStatus::instance_initialized = false;
+Logger CircuitStatus::logger = Logger("CircuitStatus");
 
-int8_t CircuitStatusModule::init() {
-    return CircuitPeriphery::init();
+void CircuitStatus::init() {
+    CircuitPeriphery::init();
+    mode = Module::Mode::OPEARTIONAL;
 }
 
-void CircuitStatusModule::spin_once() {
+void CircuitStatus::spin_once() {
     static uint32_t next_temp_pub_ms = 1000;
```

Comment:
We don't need next_temp_pub_ms and next_status_pub_ms because we specify the frequency in the constructor